src/js/live-tracker.js

```diff
@@ -257,33 +296,55 @@ class LiveTracker extends Component {
   /**
    * Determines if currentTime is at the live edge and won't fall behind
    * on each seekableendchange
+   *
+   * @return {boolean}
+   *         Wether playback is at the live edge
    */
   atLiveEdge() {
     return !this.behindLiveEdge();
   }
 
   /**
    * get what we expect the live current time to be
+   *
+   * @return {number}
+   *         The expected live current time
    */
   liveCurrentTime() {
     return this.pastSeekEnd() + this.seekableEnd();
   }
 
   /**
-   * Returns how far past seek end we expect current time to be
+   * The number of seconds that have occured after seekable end
+   * changed. This will be reset to 0 once seekable end changes.
+   *
+   * @return {number}
+   *         Seconds past the current seekable end
    */
   pastSeekEnd() {
+    const seekableEnd = this.seekableEnd();
+
+    if (typeof this.lastSeekEnd_ === 'number' && seekableEnd !== this.lastSeekEnd_) {
```

Comment:
continuing from above, should lastSeekEnd_ be set to -1 on reset rather than `null`?
```suggestion
    if (this.lastSeekEnd_ !== -1 && seekableEnd !== this.lastSeekEnd_) {
```